.github/workflows/notification.yml

```diff
@@ -0,0 +1,109 @@
+name: Slack notifications for PRs and Issues
+
+on:
+  pull_request:
+    types: [opened, reopened]
+  issues:
+    types: [opened, reopened]
+
+env:
+  CHANNEL_WALLET_FEEDBACK: ${{ vars.CHANNEL_WALLET_FEEDBACK }}
+  CHANNEL_WALLET_SQUAD_BUILD: ${{ vars.CHANNEL_WALLET_SQUAD_BUILD }}
+  ON_CALL_WALLET: ${{ vars.ON_CALL_WALLET }}
+  ON_CALL_BUILD_SQUAD: ${{ vars.ON_CALL_BUILD_SQUAD }}
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    outputs:
+      skip: ${{ env.skip }}
+    steps:
+    - name: Check spam labels
+      if: ${{ contains(github.event.*.labels.*.name, 'spam') }}
+      run: |
+        echo "skip=true" >> $GITHUB_ENV
+        echo "::error:: Spam label found."
+    # - name: Check write permission
+    #   run: |
+    #     permission=$(curl -s -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/$REPO/collaborators/$USER/permission | jq -r '.permission')
+    #     if [ "$permission" != "admin" ] && [ "$permission" != "write" ]; then
+    #       echo "::notice:: User $USER has $permission permission."
+    #       exit 0
+    #     fi
+    #     echo "skip=true" >> $GITHUB_ENV
+    #     echo "::error:: User $USER has $permission permission. No slack alert required."
+    #   env:
+    #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    #     REPO: ${{ github.repository }}
+    #     USER: ${{ github.actor }}
```

Comment:
do we need this commented code still?

Reply:
@cb-jake yes. i should have left some comments there...
this is to skip alerting when one of us opens a PR. will be reenabled after testing.

Reply:
https://coinbase.slack.com/archives/C04M9D8D41W/p1675977215650019